test.js

```diff
@@ -0,0 +1,102 @@
+var expect = require('chai').expect;
+var gulpUtil = require('gulp-util');
+
+var gulpEval = require('./');
+
+var moduleContent =
+`module.exports = {
+    block: 'page',
+    content: 'hello'
+}`;
+
+var evalContent =
+`({
+    block: 'page',
+    content: 'hello'
+})`;
+
+it('should eval simple expression', function(done) {
+    var stream = gulpEval();
+
+    stream.on('data', function(file) {
+        expect(file.data).to.exist;
+    });
+    stream.on('end', done);
+
+    stream.write(new gulpUtil.File({
+        path: 'file.js',
+        contents: new Buffer('({})')
+    }));
+
+    stream.end();
+});
+
+it('should not eval simple object', function(done) {
+    var stream = gulpEval();
+
+    stream.on('data', function(file) {
+        expect(file.data).not.to.exist;
+    });
+    stream.on('end', done);
+
+    stream.write(new gulpUtil.File({
+        path: 'file.js',
+        contents: new Buffer('{}')
+    }));
```

Comment:
https://nodejs.org/api/stream.html#stream_writable_write_chunk_encoding_callback
Returns: <Boolean> true if the data was handled completely.